dataprofiler/tests/data_readers/test_parquet_data.py

```diff
@@ -1,5 +1,6 @@
 import os
 import unittest
+from io import BytesIO, StringIO
```

Comment:
Why did you import StringIO but there is no string stream test? Is there going to be one? Why isn't there one for Parquet and Avro?